druid-shell/src/backend/windows/window.rs

```diff
@@ -1488,7 +1489,7 @@ impl WindowBuilder {
                 pos_y,
                 width,
                 height,
-                0 as HWND,
+                parent_hwnd.unwrap_or_else(|| 0 as HWND),
```

Comment:
I think `unwrap_or` here would be simpler than `unwrap_or_else`

Reply:
Done